[2023-06-10T02:21:42.500+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: fhv_taxi_data.download_dataset_task scheduled__2022-01-02T07:00:00+00:00 [queued]>
[2023-06-10T02:21:42.518+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: fhv_taxi_data.download_dataset_task scheduled__2022-01-02T07:00:00+00:00 [queued]>
[2023-06-10T02:21:42.518+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-06-10T02:21:42.519+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2023-06-10T02:21:42.519+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-06-10T02:21:42.552+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-01-02 07:00:00+00:00
[2023-06-10T02:21:42.556+0000] {standard_task_runner.py:54} INFO - Started process 5023 to run task
[2023-06-10T02:21:42.564+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'fhv_taxi_data', 'download_dataset_task', 'scheduled__2022-01-02T07:00:00+00:00', '--job-id', '246', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_multiFile_v02.py', '--cfg-path', '/tmp/tmpj8js702k']
[2023-06-10T02:21:42.570+0000] {standard_task_runner.py:83} INFO - Job 246: Subtask download_dataset_task
[2023-06-10T02:21:42.572+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/data_ingestion_gcs_multiFile_v02.py
[2023-06-10T02:21:42.655+0000] {data_ingestion_gcs_multiFile_v02.py:20} INFO - dtc_data_lake_dtc-dezc-001 is the bucket.
[2023-06-10T02:21:42.690+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-06-10T02:21:42.758+0000] {task_command.py:384} INFO - Running <TaskInstance: fhv_taxi_data.download_dataset_task scheduled__2022-01-02T07:00:00+00:00 [running]> on host f41d61956f54
[2023-06-10T02:21:42.821+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-06-10T02:21:42.859+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=fhv_taxi_data
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-02T07:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-02T07:00:00+00:00
[2023-06-10T02:21:42.860+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-10T02:21:42.860+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'curl -sSLf https://github.com/DataTalksClub/nyc-tlc-data/releases/tag/fhv/fhv_tripdata_2022-01.csv.gz > /opt/***/fhv_tripdata_2022-01.csv.gz']
[2023-06-10T02:21:42.870+0000] {subprocess.py:86} INFO - Output:
[2023-06-10T02:21:43.521+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-10T02:21:43.543+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=fhv_taxi_data, task_id=download_dataset_task, execution_date=20220102T070000, start_date=20230610T022142, end_date=20230610T022143
[2023-06-10T02:21:43.585+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-06-10T02:21:43.604+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-06-11T01:52:42.560+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: fhv_taxi_data.download_dataset_task scheduled__2022-01-02T07:00:00+00:00 [queued]>
[2023-06-11T01:52:42.566+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: fhv_taxi_data.download_dataset_task scheduled__2022-01-02T07:00:00+00:00 [queued]>
[2023-06-11T01:52:42.567+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-06-11T01:52:42.568+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2023-06-11T01:52:42.568+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-06-11T01:52:42.575+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-01-02 07:00:00+00:00
[2023-06-11T01:52:42.579+0000] {standard_task_runner.py:54} INFO - Started process 8129 to run task
[2023-06-11T01:52:42.582+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'fhv_taxi_data', 'download_dataset_task', 'scheduled__2022-01-02T07:00:00+00:00', '--job-id', '799', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_multiFile_v02.py', '--cfg-path', '/tmp/tmpp_pccts5']
[2023-06-11T01:52:42.583+0000] {standard_task_runner.py:83} INFO - Job 799: Subtask download_dataset_task
[2023-06-11T01:52:42.584+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/data_ingestion_gcs_multiFile_v02.py
[2023-06-11T01:52:42.642+0000] {data_ingestion_gcs_multiFile_v02.py:20} INFO - dtc_data_lake_dtc-dezc-001 is the bucket.
[2023-06-11T01:52:42.657+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-06-11T01:52:42.697+0000] {task_command.py:384} INFO - Running <TaskInstance: fhv_taxi_data.download_dataset_task scheduled__2022-01-02T07:00:00+00:00 [running]> on host f41d61956f54
[2023-06-11T01:52:42.719+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-06-11T01:52:42.745+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=fhv_taxi_data
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-02T07:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-02T07:00:00+00:00
[2023-06-11T01:52:42.746+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-11T01:52:42.747+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'curl -sSLf https://github.com/DataTalksClub/nyc-tlc-data/releases/download/fhv/fhv_tripdata_2022-01.csv.gz > /opt/***/fhv_tripdata_2022-01.csv.gz']
[2023-06-11T01:52:42.757+0000] {subprocess.py:86} INFO - Output:
[2023-06-11T01:52:43.037+0000] {subprocess.py:93} INFO - curl: (22) The requested URL returned error: 404
[2023-06-11T01:52:43.052+0000] {subprocess.py:97} INFO - Command exited with return code 22
[2023-06-11T01:52:43.082+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 22.
[2023-06-11T01:52:43.088+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=fhv_taxi_data, task_id=download_dataset_task, execution_date=20220102T070000, start_date=20230611T015242, end_date=20230611T015243
[2023-06-11T01:52:43.106+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 799 for task download_dataset_task (Bash command failed. The command returned a non-zero exit code 22.; 8129)
[2023-06-11T01:52:43.131+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-06-11T01:52:43.150+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
